### PR TITLE
Adding 2 themes!

### PR DESCRIPTION
**Pull request by fimion**
This will add 2 themes created on my stream, I hope you enjoy them (and maybe feel a bit queasy)!

### CODE REVIEW

File: src/themes/alex-party.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+[data-twitch-emote] {
+	width: 1.25em;
+}
+
+[data-twitch-mentioned-user] {
+	background-color: transparent;
+	font-family: 'Papyrus', 'Comic Sans MS', cursive;
+	font-weight: 600;
+	/* -webkit-text-stroke: 1px var(--twitch-purple); */
```
Comment (BenDMyers) [inline src/themes/alex-party.css:86]: Would you be willing to nuke this comment?

File: src/themes/retro-swirly.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+[data-twitch-message] .twitch-chat-message {
+	display: inline;
+}
+
+[data-twitch-emote] {
+	width: 1.2em;
+}
+
+[data-twitch-mentioned-user] {
+	background-color: #dedee3;
```
Comment (BenDMyers) [inline src/themes/retro-swirly.css:158]: Just checking - do we want to leave the mentions with this gray background? It's totally cool if yes, just want to make sure.

File: src/themes/alex-party.css (hunk 1 above)
Comment (fimion) [reply src/themes/alex-party.css]: killed it.

File: src/themes/retro-swirly.css (hunk 2 above)
Comment (fimion) [reply src/themes/retro-swirly.css]: I actually redid the styles a little bit on this one. the contrast on the user text against a light blue background wasn't great. So I updated it to use cards.

Comment (BenDMyers) [reply src/themes/retro-swirly.css]: Ahh, I like the card treatment! 👍🏻

**Review by BenDMyers — APPROVED** [review]

**Comment by BenDMyers** [comment]
@all-contributors Please add @fimion for content and design!

**Pull request merged**